src/Generator/ValueGenerator.php

```diff
@@ -311,7 +312,7 @@ public function getAutoDeterminedType($value)
                         return self::TYPE_CONSTANT;
                     }
 
-                    if (strpos($value, $constant) !== false) {
+                    if (str_contains($value, $constant)) {
```

Comment:
Yeah, probably to baseline...

Reply:
i'he try to find a solution but : 

```
ERROR: TooManyTemplateParams - src/Generator/ValueGenerator.php:70:13 - Laminas\Stdlib\ArrayObject<array-key, string> has too many template params, expecting 0 (see https://psalm.dev/184)
     * @var SplArrayObject<array-key, string>|StdlibArrayObject<array-key, string>
```

Reply:
I think a suppression is OK here: no need to add further complexity in internal types :thinking: